packages/monaco/src/browser/monaco-command.ts

```diff
@@ -46,6 +46,7 @@ export namespace MonacoCommands {
 
     export const EXCLUDE_ACTIONS = new Set([
         'editor.action.quickCommand',
+        'editor.action.toggleStickyScroll', // Handled by `editor` package.
```

Comment:
Would it be an option to remove our implementation of the command and go with the monaco implementation? It seems like this command is tightly bound to the monaco editor.

Reply:
I agree that sticky scroll isn't really a 'generic' editor capability. On the other hand, we do register the sticky scroll preference in the `editor` package, and as I say in the description, the main effect of both implementations is a change in that preference, so from that perspective, it has a natural home there.